src/data/object_events/object_event_graphics_info_pointers.h

```diff
@@ -392,7 +392,7 @@ const struct ObjectEventGraphicsInfo *const gObjectEventGraphicsInfoPointers[NUM
     [OBJ_EVENT_GFX_HO_OH]                    = &gObjectEventGraphicsInfo_HoOh,
     [OBJ_EVENT_GFX_CELEBI]                   = &gObjectEventGraphicsInfo_Celebi,
     [OBJ_EVENT_GFX_KABUTO]                   = &gObjectEventGraphicsInfo_Kabuto,
-    [OBJ_EVENT_GFX_DEOXYS_D]                 = &gObjectEventGraphicsInfo_DeoxysD,
+    [OBJ_EVENT_GFX_TRUCK]                 = &gObjectEventGraphicsInfo_DeoxysD,
```

Comment:
Keeping everything else as deoxys D here is kinda cursed but if we must :V

Reply:
I'm sure I'll deal with it at some point - but you're the one who needs to review all 90 trillion files that get edited :p